testfiles/1ZoneUncontrolled.idf

```diff
@@ -364,6 +364,10 @@
 
   Output:Variable,*,Site Outdoor Air Drybulb Temperature,hourly;
 
+  Output:Variable,*,Site Total Sky Cover,hourly;
+
+  Output:Variable,*,Site Opaque Sky Cover,hourly;
+
```

Comment:
Ideally these would be added to the develop branch in a separate PR so you could see the diffs.  But that's a lot of extra work, and the fix here is pretty obvious.  Either way it's good to add them here at least.

Reply:
Thanks! @Myoldmopar Since the testfiles' changes are in the last two commits, I can easily revert to the commit before the one before that (actually already did it locally and ready to push if needed). Then after merging of this one I can add back the two last ones in a new PR.

Reply:
I don't think it's necessary.  This is an obvious fix, and adding the output variables to the IDFs is a nice bonus over the unit test.  I think this is already ready to go.  CI looks good too, with just the new output variables causing "diffs".  I still need to run it locally but I think it's good.